src/python/WMCore/Database/DBFormatter.py

```diff
@@ -78,7 +78,7 @@ def formatDict(self, result):
                 for index in range(0, len(descriptions)):
                     # WARNING: Oracle returns table names in CAP!
                     if isinstance(i[index], str):
-                        entry[str(descriptions[index].lower())] = bytes(i[index])
+                        entry[str(descriptions[index].lower())] = encodeToString(i[index])
```

Comment:
This should have been enough, my bad that i missed it
```suggestion
                        entry[str(descriptions[index].lower())] = bytes(i[index], "utf-8")
```

or also

```suggestion
                        entry[str(descriptions[index].lower())] = i[index].encode("utf-8")
```